CI job fails due to missing execution permission for `tools/flutter-dependencies.sh`

### New Issue Checklist

- [X] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [X] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [X] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).
- [X] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Flutter SDK](https://github.com/parse-community/Parse-SDK-Flutter/releases). <!-- We don't investigate issues for outdated releases. -->

### Issue Description
Fixing Action error in PR #762 

Fix permission in `tools/flutter-dependencies.sh`

```shell
/home/runner/work/_temp/77210dd4-30c5-471c-[9](https://github.com/parse-community/Parse-SDK-Flutter/runs/6844657586?check_suite_focus=true#step:4:10)082-3b4ab7de3cde.sh: line 2: tools/flutter-dependencies.sh: Permission denied

Error: Process completed with exit code 126
```

### Steps to reproduce
Resume action in PR #762 

### Actual Outcome
<!-- What outcome did you get? -->

### Expected Outcome
<!-- What outcome did you expect? -->

### Environment
<!-- Be specific with versions, don't use "latest" or semver ranges like "~x.y.z" or "^x.y.z". -->

Parse Flutter SDK
- SDK version: 3.0.3
- Operating system version:   macOS 12.4

Server
- Parse Server version: 4.10

### Logs

/home/runner/work/_temp/77210dd4-30c5-471c-[9](https://github.com/parse-community/Parse-SDK-Flutter/runs/6844657586?check_suite_focus=true#step:4:10)082-3b4ab7de3cde.sh: line 2: tools/flutter-dependencies.sh: Permission denied

Error: Process completed with exit code 126.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -81,6 +81,8 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
+      - name: Make executeable flutter-dependencies.sh
+        run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: tools/flutter-dependencies.sh
       - name: Run tests

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -46,7 +46,7 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
-      - name: Make executeable flutter-dependencies.sh
+      - name: Set file permissions
         run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: |

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -46,6 +46,8 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
+      - name: Make executeable flutter-dependencies.sh
+        run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: |
           echo "publish_to: none" >> packages/flutter/pubspec.yaml

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -81,7 +81,7 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
-      - name: Make executeable flutter-dependencies.sh
+      - name: Set file permissions
         run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: tools/flutter-dependencies.sh

